tristan/ding-wall/src/components/AsideBlurb.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './AsideBlurb.css';
+
+function AsideBlurb() {
+  return (
+    <div className="aside-container">
+      <aside>
+        <h3>The house:</h3>
+        <p>Purchased in fall 2019, our house is in the midst of a major renovation.</p>
+        <p>The house is old as hell, and the walls contain a million little rodent poops. Non-dingwall-type people would have opted to tear the place down, but we decided to be all "romantic" about it, and retain what solid wood remained.</p>
+      </aside>
+    </div>
```

Comment:
Same comment as footer.js. I'll add a suggested change here to show what I mean
```suggestion
      <aside className="aside-container">
        <h3>The house:</h3>
        <p>Purchased in fall 2019, our house is in the midst of a major renovation.</p>
        <p>The house is old as hell, and the walls contain a million little rodent poops. Non-dingwall-type people would have opted to tear the place down, but we decided to be all "romantic" about it, and retain what solid wood remained.</p>
      </aside>
```